### PR TITLE
Ported  `BreakPoint` in Rust

### PR DESCRIPTION
**Pull request by itsAdee**
Looked at the implementation in JavaScript and replicated the code in RUST by going through `libadwaita`document.Works fine on my machine , please check it yourself and let me know of any problem.Thanks

### CODE REVIEW

File: src/Library/demos/Breakpoints/code.rs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+use crate::workbench;
+use adw::prelude::*;
```
Comment (Hofer-Julian) [inline src/Library/demos/Breakpoints/code.rs:2]:
```suggestion
```

**Review by Hofer-Julian — COMMENTED** [review]
Apart from the one comment, this looks good to me

**Comment by itsAdee** [comment]
> Apart from the one comment, this looks good to me

Added the suggestion :)

**Review by Hofer-Julian — APPROVED** [review]

**Comment by Hofer-Julian** [comment]
Thanks!

**Pull request merged**